drivers/flash/spi_nor.c

```diff
@@ -702,6 +702,33 @@ static int spi_nor_read(const struct device *dev, off_t addr, void *dest,
 	return ret;
 }
 
+#if defined(CONFIG_FLASH_EX_OP_ENABLED)
+static int flash_spi_nor_ex_op(const struct device *dev, uint16_t code,
+				const uintptr_t in, void *out)
```

Comment:
nit: indentation looks off, not a strong rule but it makes sense to align on the parenthesis here like [most of] the rest of the file